extract/extract.js

```diff
@@ -0,0 +1,24 @@
+var numEvents = numEvents || 1;
+
+function skipNullOrEmptyValue(key, value) {
+    return (
+        value == null ||
+        (typeof value == 'object' && Object.keys(value).length == 0)
+    ) ? undefined : value;
+}
+
+// The original find query used the same regex on the object property.
+// However, many PR events don't have "problemroulette" in that property.
+
+db.event.find({
+    raw: {
+        $regex: /problemroulette/
```

Comment:
Can the regex value be read from configuration/command line args? So that the same extract.js can be used for various queries.